api/setup/setup.go

```diff
@@ -31,6 +31,7 @@ func Schemas(ctx context.Context, management *config.ManagementContext, schemas
 	Templates(schemas)
 	ClusterRegistrationTokens(schemas)
 	addUserAction(schemas)
+	addCatalogRefreshAction(schemas)
```

Comment:
I'd prefer these two methods to be Users(schemas) and Catalogs(schemas) to match the style above

Reply:
👍